cli/command/trust/key_generate.go

```diff
@@ -0,0 +1,87 @@
+package trust
+
+import (
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/trust"
+	"github.com/docker/notary"
+	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/tuf/data"
+	tufutils "github.com/docker/notary/tuf/utils"
+	"github.com/spf13/cobra"
+)
+
+func newKeyGenerateCommand(dockerCli command.Streams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "key-generate NAME [NAME...]",
+		Short: "Generate a signing key",
```

Comment:
non-blocking: Perhaps we should call it a "signing key-pair" or some such?

Reply:
Also,"Generate and load" feels more true, WDYT?

Reply:
@ashfall: sounds good for both!